staging/src/k8s.io/apimachinery/pkg/util/yaml/decoder.go

```diff
@@ -92,6 +92,8 @@ type YAMLDecoder struct {
 // the caller in framing the chunk.
 func NewDocumentDecoder(r io.ReadCloser) io.ReadCloser {
 	scanner := bufio.NewScanner(r)
+	buf := make([]byte, 64*1024)
```

Comment:
suggest 4k to match existing default buffer size

Reply:
+1